client/src/ingredients/specifications.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+
+export function Specifications({ specifications }) {
+  if (specifications === "non-standard") {
+    return (
+      <>
+        <h2>Non-standard feature</h2>
+        <p>
+          This feature is non-standard and is not included in any official
+          specification. Do not use it on production sites facing the Web: it
+          will not work for every user. There may also be large
+          incompatibilities between implementations and the behavior may change
+          in the future.
+        </p>
+      </>
+    );
+  } else {
+    return (
+      <>
+        <h2>Specifications</h2>
+        <ul>
```

Comment:
What if the `specifications` is an array and what it it's empty?? Then you end up with 

```html
<h2>Specifications</h2>
<ul>
</ul>
```

I think, on line 18 before the `return` do something like...
```javascript
if (!specifications.length) {
  throw new Error('specifications is an empty array');
}
```